sdk/monitor/arm-monitor/test/monitor_examples.ts

```diff
@@ -160,8 +161,8 @@ describe("Monitor test", () => {
     }, testPollingOptions)
     workspaceId = workspace.id || "";
   });
-  // need ask mary or qiaoqiao to add "Storage Blob Data Contributor" permission before create eventhub
-  it("eventhub create test", async function () {
+  // skip this case as no data plane write permissions
+  it.skip("eventhub create test", async function () {
```

Comment:
pelase make sure to create issue to track this.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/30204